### ISSUE TITLE
No menu item labels after PR #1728 on Windows

### ISSUE THREAD

**Issue by stnkl** [getferdi/ferdi#1746]
**Bug description:**
After #1728 some menu items don't have any text anymore. According to the documentation it should receive its text automatically for each `role`. However this doesn't seem to be the case on Windows for some reason. I assume this works as expected on MacOS because it must have worked for @vraravam when performing those changes?

Also pasting from the "Edit" menu still doesn't work (on Windows at least). But that's another issue (#1726).

**Steps to reproduce:**
Open any of the menus on Windows.

**Expected behavior**
Menu items should have labels.

**Screenshots**

![grafik](https://user-images.githubusercontent.com/17520641/128395601-08e52fbd-abff-4017-a145-0b177ef3dfbe.png)

![grafik](https://user-images.githubusercontent.com/17520641/128396084-6b15fc63-5ef8-4cf7-9f7d-4289f3e011e5.png)

![grafik](https://user-images.githubusercontent.com/17520641/128396357-2d0eae5e-f74b-4e8b-85ee-23882c3d840e.png)

**Environment:**

- Operating System: Windows 20H2
- Ferdi Version: 5.6.1-nightly.14
- Server: Franz

- Debug information:

**Comment by vraravam** [comment]
<img width="220" alt="Screenshot 2021-08-06 at 5 11 02 AM" src="https://user-images.githubusercontent.com/69629/128434846-80b22612-0b8b-47d4-8b68-99aa80a4640b.png">

Yes - the menu items show perfectly well on macos. Let me undo that change partially (resurrect the labels, but keep the de-duplication still committed)

**Comment by vraravam** [comment]
This should be resolved in the nightly 16 - which is in the process of building/publishing right now. Will release in the next 20 mins or so

**Comment by stnkl** [comment]
I can confirm it is working again on Windows. Thank you!
